### DISCUSSION TITLE
How disable auto backup file .bkp file

### DISCUSSION THREAD

**Discussion by guohaison** [jgraph/drawio-desktop#1003] [Q&A] [answered]
How disable auto backup file .$draw1.png.bkp file,  every edit and save draw1.png,  generation bkp file

**Comment by davidjgraph** [comment]
What version are you using?

**Reply by guohaison** [reply]
18.0.4

**Comment by guohaison** [comment]
18.0.4 (draw.io-18.0.4-windows-no-installer.exe)

**Comment by m-mohamedin** [comment, marked as answer]
An option to disable automatic backup would be added to the next release (Extras -> Automatic Backup)

**Comment by xtcamille** [comment]
How do I disable auto backup file .bkp file in version 21.2.8?

**Reply by davidjgraph** [reply]
Did you look in the menu under extras->automatic backup as the selected answer suggests?

**Reply by Ynjxsjmh** [reply]
@davidjgraph That doesn't help, it only disables automatic backup. When I use `Ctrl-s` to save the diagram, I could still see `.$draw1.png.dbkp` file. I use Dropbox to sync my file, so I don't need the backup ability which would lead Dropbox upload and delete the dbkp file all the time while I edit the diagram. I use v22.1.2 on Windows.
